main.go

```diff
@@ -209,31 +215,35 @@ func getAPISpecs() []*APISpec {
 			"prefix": "main",
 		}).Debug("Using RPC Configuration")
 
-		apiSpecs = APILoader.LoadDefinitionsFromRPC(config.SlaveOptions.RPCKey)
+		APISpecs = APILoader.LoadDefinitionsFromRPC(config.SlaveOptions.RPCKey)
 	} else {
-		apiSpecs = APILoader.LoadDefinitions(config.AppPath)
+		APISpecs = APILoader.LoadDefinitions(config.AppPath)
 	}
 
 	log.WithFields(logrus.Fields{
 		"prefix": "main",
-	}).Printf("Detected %v APIs", len(apiSpecs))
+	}).Printf("Detected %v APIs", len(APISpecs))
 
 	if config.AuthOverride.ForceAuthProvider {
-		for i := range apiSpecs {
-			apiSpecs[i].AuthProvider = config.AuthOverride.AuthProvider
+		for i := range APISpecs {
+			APISpecs[i].AuthProvider = config.AuthOverride.AuthProvider
 		}
 	}
 
 	if config.AuthOverride.ForceSessionProvider {
-		for i := range apiSpecs {
-			apiSpecs[i].SessionProvider = config.AuthOverride.SessionProvider
+		for i := range APISpecs {
+			APISpecs[i].SessionProvider = config.AuthOverride.SessionProvider
 		}
 	}
-
-	return apiSpecs
 }
 
-func getPolicies() {
+var Policies = map[string]Policy{}
+var policySyncMux sync.Mutex
```

Comment:
Same as my other `Mutex` question - certain that locking only on writes is enough?